src/TConsole.cpp

```diff
@@ -2423,7 +2423,6 @@ void TConsole::createMapper( int x, int y, int width, int height )
         mpHost->mpMap->mpM = mpMapper->glWidget;
         mpHost->mpMap->mpHost = mpHost;
         mpHost->mpMap->mpMapper = mpMapper;
-        mpMapper->mpHost = mpHost;
```

Comment:
Is this not needed because it is done by the `dlgMapper` constructor? :confused: